src/query_generate.rs

```diff
@@ -28,53 +28,53 @@ pub fn generate_query_code(table_name: &str, rows: &[TableColumn]) -> String {
     query_code.push_str(&format!("impl {}Set {{\n", struct_name));
 
     // Generate query code for SELECT statements
-    query_code.push_str(&generate_select_query_code(table_name, schema_name, rows));
+    query_code.push_str(&generate_select_query_code(table_name, schema_name, &rows));
     query_code.push('\n');
 
     // Generate query code for SELECT BY PK statements
     query_code.push_str(&generate_select_by_pk_query_code(
         table_name,
         schema_name,
-        rows,
+        &rows,
     ));
     query_code.push('\n');
 
     // Generate query code for SELECT MANY BY PK statements
     query_code.push_str(&generate_select_many_by_pks_query_code(
         table_name,
         schema_name,
-        rows,
+        &rows,
     ));
     query_code.push('\n');
 
     // Generate query code for SELECT BY PK Optional statements
     query_code.push_str(&generate_select_by_pk_query_code_optional(
         table_name,
         schema_name,
-        rows,
+        &rows,
     ));
     query_code.push('\n');
 
-    query_code.push_str(&generate_unique_query_code(table_name, schema_name, rows));
-    query_code.push('\n');
+    // query_code.push_str(&generate_unique_query_code(table_name, schema_name, &rows));
```

Comment:
This created duplicate function names for some of my tables. I had to comment on this and forget about it. will look into it soon.

Reply:
Maybe it's because primary keys are unique - the `by_{}` one is meant to be for the pk, where as this is meant to be for all other unique keys.

Reply:
We're able to change this to whatever else makes sense - though it would be good to have some indication of what are the primary key queries and what are the other queries.

Reply:
PK query is sort of important for cases where the PK is two or more columns, which may or may not each be unique

Reply:
I guess I fixed it by filtering more based on the given table name and unique columns.